### PR TITLE
[Counterproposal] Rename working_directory to project_directory and always use it as input for executable

### PR DESCRIPTION
**Pull request by rcannood**
Counter-proposal to #16 

Changes:

* `pro/generate-documentation-qmd`: Renamed `working_directory` to `project_directory`, and renamed `input_dir` to `src` (#17).

### CODE REVIEW

File: pro/generate-documentation-qmd/README.md [hunk 1]
```diff
@@ -1,45 +1,44 @@
...
+  `latest`.
+- `src`: *(Optional)* An override for the `--src` parameter in
+  `viash ns list`. Example: `src/`.
+
+### Outputs
 
 ## Examples
 
-```yaml
+``` yaml
```
Comment (DriesSchaumont) [inline pro/generate-documentation-qmd/README.md:41]: Idk if this space is required :)

**Review by DriesSchaumont — CHANGES_REQUESTED** [review]
ALGTM :) Will now try and test this

File: pro/generate-documentation-qmd/README.md [hunk 2]
```diff
@@ -1,45 +1,44 @@
 # generate-documentation-qmd
 
-Create quarto markdown files for viash components or workflows, based on a jinja template.
-
-## Usage
-
-This action will look for viash configs for workflows and components in a directory,
```
[the remark this answers is not in the document]
Comment (rcannood) [reply pro/generate-documentation-qmd/README.md:7]: I added it back!







... Mostly

File: pro/generate-documentation-qmd/README.md (hunk 1 above)
Comment (rcannood) [reply pro/generate-documentation-qmd/README.md:48]: This is auto-generated by Quarto when using the gfm format.

**Review by DriesSchaumont — APPROVED** [review]

**Pull request merged**